script_automation/esa_ubuntu_k3_single/_index.md

```diff
@@ -0,0 +1,45 @@
+# This example can be used to install ESA on an Ubuntu system with K3s. 
+
+This example requires an Ubuntu machine or VM running K3s. We will Arc Connect the system through this example. 
```

Comment:
If you have prerequisites, create a prerequisites section and add bullet points for each prereq with a link to the guide on how to install it